app/views/health/patient/_engagement.haml

```diff
@@ -5,12 +5,16 @@
 .patient__engagement.mr-2.text-right
   - if ! patient.engaged?
     - if patient.days_to_engage.zero?
-      .patient__engagement-expired
-        PCTP signature is overdue!
+      - if patient.careplans.active.recent.exists?
```

Comment:
maybe `! patient.careplans.active.recent.pcp_signed.exists?` instead? It seems like there are a lot of ways `engaged?` can return false, so I'd rather default to incomplete rather than unsigned.